doc/integration.rst

```diff
@@ -68,6 +71,12 @@ Parameters
 +-----------------------+-------------+------------+-----------------------------------------------------------------+
 | Name                  | Type/Range  | Default    | Description                                                     |
 +=======================+=============+============+=================================================================+
+| ``PMPEnable``         | bit         | 0          | Enable PMP checking                                             |
```

Comment:
I would change `Enable PMP checking` to `Enable PMP support`. The reason is that just because you instantiate the PMP module, it is not necessarily configured and active (as long as you stay in M-mode and don't lock the entries, anything is allowed). We had a similar discussion for the performance counters.

Reply:
done